website/www/site/data/en/join_beam.yaml

```diff
@@ -11,15 +11,20 @@
 # limitations under the License.
 
 - list:
-    - Join Slack
-    - Follow us on Twitter to get the latest information
-    - Follow us on YouTube to know first about the latest tutorials
-    - Follow meetup closest to your city
+    - { title: Mailing Lists, description: Join the official Apache Beam user and developer mailing lists. Learn how <a href="https://beam.apache.org/community/contact-us/">here</a> }
+    - { title: Slack Channel, description: "For real-time communication, join us on the #beam channels in the ASF workspace." }
+    - { title: Twitter account, description: Follow @ApacheBeam on Twitter for the most recent news on the project and community! }
+    - { title: YouTube channel, description: Follow our YouTube channel for the latest training and use cases. }
+    - { title: StackOverflow, description: Ask your Beam questions in StackOverflow using the apache-beam tag. }
+    - { title: Events, description: Follow our calendar in the homepage which has the latest events organized by the community. }
```

Comment:
Please add links for 
- [ ] the-asf Slack
- [ ] @ApacheBeam Twitter
- [ ] Apache Beam YouTube
- [ ] StackOverflow apache-beam tag
- [ ] Event calendar.

Reply:
done